src/spyglass/histology/v1/histology.py

```diff
@@ -0,0 +1,141 @@
+import datajoint as dj
+
+from spyglass.common import (  # noqa: F401
+    AnalysisNwbfile,
+    CoordinateSystem,
+    LabMember,
+    Subject,
+)
+from spyglass.utils import SpyglassMixin, logger
+
+schema = dj.schema("histology_v1")
+
+
+@schema
+class Histology(SpyglassMixin, dj.Manual):
+    definition = """
+    # Represents a single histology preparation for a subject
+    -> Subject
+    histology_id: varchar(32)           # User-defined ID (e.g., 'probe_track_run1')
+    ---
+    prep_date=NULL: date                # Optional: Date of tissue preparation
+    slice_orientation: enum(            # Orientation of sections
+        "coronal",
+        "sagittal",
+        "horizontal",
+        "other"
+    )
+    slice_thickness: float              # (um) Thickness of sections
+    mounting_medium=NULL: varchar(128)  # e.g., 'DPX', 'Fluoromount-G'
+    experiment_purpose: varchar(1024)   # e.g., 'Probe track recovery', 'Anatomical ref'
+    notes="": varchar(2048)             # Optional general notes about the preparation
+    -> [nullable] LabMember.proj(histology_experimenter='user_name') # Optional: who did the prep?
+
+    # --- Data Source ---
+    output_format='TIFF stack': varchar(64) # Format of raw image data from scanner
+    raw_scan_path: varchar(512)         # Path to the raw output (e.g., folder containing TIFF stack)
+    """
+
+    class HistologyStain(SpyglassMixin, dj.Part):
+        definition = """
+        # Details of specific stains used in a histology preparation
+        -> Histology
+        stain_index: tinyint unsigned    # Use index for multiple stains (0, 1, 2...)
+        ---
+        identified_feature: varchar(128) # Biological target/marker (e.g., 'Nissl Bodies', 'ChR2-tdTomato+', 'ProbeTrack_DiI')
+        visualization_agent: varchar(128)# Method/molecule making feature visible (e.g., 'Cresyl Violet', 'Native tdTomato', 'DiI', 'Alexa 488')
+        stain_type: enum(                # Type of staining method used
```

Comment:
Is this list exhaustive? Would we want to make this a varchar for flexibility?

Reply:
If you're not sure, then yes, I'd convert